[18:32:19][Step 2/3] [ERROR]   The project org.wildfly:wildfly-ts-integ-picketlink:9.0.0.CR1-SNAPSHOT (/opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/picketlink/pom.xml) has 1 error
[18:32:19][Step 2/3] [ERROR]     'dependencies.dependency.version' for rhino:js:jar must be a valid version but is '${version.rhino.js}'. @ org.wildfly:wildfly-ts-integ-picketlink:[unknown-version], /opt/buildAgent/work/a31d203e70e89f90/testsuite/integration/picketlink/pom.xml, line 52, column 30
[18:32:19][Step 2/3] [ERROR]

--- a/pom.xml
+++ b/pom.xml
@@ -217,6 +217,7 @@
         <version.org.yaml.snakeyaml>1.15</version.org.yaml.snakeyaml>
         <version.sun.jaxb>2.2.11</version.sun.jaxb>
         <version.sun.saaj-impl>1.3.16-jbossorg-1</version.sun.saaj-impl>
+        <version.rhino.js>1.7R2</version.rhino.js>
         <version.wsdl4j>1.6.3</version.wsdl4j>
         <version.xml-resolver>1.2</version.xml-resolver> <!-- Apache xml-resolver -->
         <version.xom>1.2.10</version.xom>

